update_prevalence.py

```diff
@@ -830,7 +830,7 @@ def rollup_population(parent: Place, child_attr: str) -> None:
                 for child in children.values():
                     parent.population += child.population
                 if not parent.population and parent.name not in fake_names:
```

Comment:
Could you add Antarctica to fake_names (and perhaps rename fake_names to something like "places_without_population")? Otherwise we'll be spamming ourselves with error logs.

Reply:
I don't think we want to do that without other changes. That would stop us from throwing for Antarctica, but Antarctica would stay in the list of countries with whatever partial data we calculated. Maybe not a big deal?

What I'd suggest instead is to create a list of `countries_to_ignore` and just purge them in advance of any calculations, so we don't raise any errors in the first place. Or we could rename `fake_names` to `names_to_ignore`, add Antarctica, and then do that same pass through the countries list to remove all the "fake names" before we do any calculations.

For simplicity, and in order to get this merged asap, I'd like to do that in a follow-up PR. Sound good?